server/shared/publishing/helpers.js

```diff
@@ -178,9 +194,9 @@ function fetchDefaultContainer(container) {
     }));
 }
 
-function fetchCustomContainers(parent) {
+function fetchCustomContainers(parent, types) {
```

Comment:
Would it make sense to inherit it from configuration based on parent type (rather than passing additional params)

Reply:
I am not sure that it would. Both this method for fetching custom containers and the method for fetching the default containers get their types from the fetchContainers method. So, if we remove it from one of them we need to remove it from the other one. And I do not see what do we gain by doing that.